web/test/pages/profile.test.tsx

```diff
@@ -1813,6 +1819,23 @@ describe("profile", () => {
     });
   });
 
+  it("resets naicsCode task when the industry is changed and page is saved", async () => {
+    const userData = generateUserData({
+      profileData: generateProfileData({ industryId: "cosmetology" }),
+      taskProgress: {
+        [naicsCodeTaskId]: "COMPLETED",
+      },
+    });
+    renderPage({ userData });
+    chooseTab("info");
+    selectByValue("Industry", "e-commerce");
+    clickSave();
+
+    await waitFor(() => {
+      expect(currentUserData().taskProgress[naicsCodeTaskId]).toEqual("NOT_STARTED");
+    });
+  });
+
```

Comment:
I know we are checking that the NAICS code is being reset in the OnboardingIndustry.test.tsx but it might make it more readable in the future if we are explicitly checking it here as well so we see that two changes are expected to happen? Just a thought.